media/pages/sk/ochrana-osobnych-udajov/page.html

```diff
@@ -21,17 +21,13 @@ <h1>Ochrana osobných údajov</h1>
 </p>
 <p>
 týmto v zmysle ustanovenia § 15 Zákona poskytuje informácie o skutočnostiach a poučenie o právach,
-týkajúcich sa osobných údajov dotknutej osoby:
-</p>
-<p>
-nasledovne:
+týkajúcich sa osobných údajov dotknutej osoby nasledovne:
```

Comment:
@martinmacko47 Nie je to typo? Mne sa velmi nepacilo ze to bolo takto oddelene v dvoch riadkoch a 2-krat dvojbodka.

Reply:
Ano, bola to chyba. Vdaka, ze si to opravil.